Frameworks/MetalPetal/Filters/MTIMPSImageConvolution.h

```diff
@@ -0,0 +1,30 @@
+//
+//  MTIMPSImageConvolution.h
+//  Pods
+//
+//  Created by shuyj on 2017/8/14.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "MTIFilter.h"
+
+@interface ConvolutionInputSets : NSObject<NSCopying>
```

Comment:
This should be a private class, because it is not exposed via any public interfaces.